sstmac/libraries/pthread/sstmac_sched.h

```diff
@@ -73,14 +73,10 @@ extern int SSTMAC_sched_getscheduler (pid_t pid);
 extern int SSTMAC_sched_yield (void);
 
 /* Get maximum priority value for a scheduler.  */
-static inline int SSTMAC_sched_get_priority_max (int  /*algorithm*/){
-  return 99;
-}
+extern int SSTMAC_sched_get_priority_max(int algorithm);
```

Comment:
Um what is the point of `extern` on a function?

Reply:
I'm leaving the `extern` to match the rest of the file for now. It doesn't hurt anything or create warnings.